### PR TITLE
Created headline

### PR DESCRIPTION
**Pull request by akbari4yaseen**
Headline of web page and social media contact added

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -4,8 +4,42 @@
...
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="brand">
+        <h1 class="brand-name">Brand Name</h1>
+      </div>
```
Comment (vechicin) [inline portfolio.html]: This should be an `<a>` tag that says `My Logo`.

Screen Shot 2022-08-18 at 8 37 50

Please fix it.

File: portfolio.html [hunk 2]
```diff
@@ -4,8 +4,42 @@
...
+      <div class="hamburger-btn">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
```
Comment (vechicin) [inline portfolio.html]: This content should be wrapped in a `<nav>` tag.

Screen Shot 2022-08-18 at 8 38 57

Please fix it.

File: style.css [hunk 3]
```diff
@@ -1,10 +1,78 @@
...
+  background-color: #fff;
   font-family: "Poppins", sans-serif;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+}
```
Comment (vechicin) [inline style.css]: This section is not matching the template design.

Screen Shot 2022-08-18 at 8 40 18

Screen Shot 2022-08-18 at 8 40 31

Please fix it.

Screen Shot 2022-08-18 at 8 40 51

File: style.css [hunk 4]
```diff
@@ -1,10 +1,78 @@
...
+  background-color: var(--secondary-color);
+}
+
+main {
+  padding: 10px 30px;
+}
+
+.headline {
+  margin-top: 40px;
+}
```
Comment (vechicin) [inline style.css]: This section doesn't have the template's background image, please add it.

Screen Shot 2022-08-18 at 8 42 12

Additionally, this content should be aligned with flexbox.

Screen Shot 2022-08-18 at 8 45 29

File: portfolio.html [hunk 5]
```diff
@@ -4,8 +4,42 @@
...
+          medical, finance, and many more.
+        </p>
+        <p>If you are interested, please feel free to contact me.</p>
+        <h2>LET’S CONNECT</h2>
+        <div class="social-media">
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+          <a href="#"><i class="fa-brands fa-square-facebook"></i></a>
+        </div>
```
Comment (vechicin) [inline portfolio.html]: This content should be wrapped in a `<ul>` tag with each icon wrapped in a`<li>` tag.

Screen Shot 2022-08-18 at 8 44 09

File: portfolio.html [hunk 6]
```diff
@@ -4,8 +4,42 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
```
Comment (vechicin) [inline portfolio.html:10]: Your project looks very dissimilar from the template design.

Screen Shot 2022-08-18 at 8 47 37

Screen Shot 2022-08-18 at 8 47 54

Please add all the necessary details to make it as identical as possible.

Screen Shot 2022-08-18 at 8 48 25

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @akbari4yaseen ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Linters checks passing.

## Required Changes ♻️

- [x] Please submit **ONLY ONE PR** moving forward. I only reviewed this PR as reviewing two separate PRs for one milestone is tremendously cumbersome for us reviewers.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html (hunk 6 above)
Comment (akbari4yaseen) [reply portfolio.html:10]: All the necessary details added, and also bug fixed

**Comment by akbari4yaseen** [comment]
Hi @vechicin please review again.

File: portfolio.html [hunk 7]
```diff
@@ -7,5 +7,60 @@
...
+        </div>
+        <nav>
+          <div class="menu-icon">
+            <img src="images/Union.png" alt="menu button" />
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="headline">
+          <h2 class="title">I’m Yaseen Glad to see you!</h2>
```
Comment (Ismailco) [inline portfolio.html:24]: - [ ] [OPTIONAL] This header should be an `<h1>` kindly consider changing it so your page has one `<h1>` tag

File: portfolio.html [hunk 8]
```diff
@@ -7,5 +7,60 @@
...
+          <p class="details">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p></p>
+          <h2 class="contact-title">LET’S CONNECT</h2>
```
Comment (Ismailco) [inline portfolio.html:32]: - [ ] [OPTIONAL] It's better if you convert this header into a link so you can link to external websites or add a link to send an email

**Review by Ismailco — APPROVED** [review]
### Hi @akbari4yaseen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**